plugins/arElasticSearchPlugin/lib/model/arElasticSearchActorPdo.class.php

```diff
@@ -262,6 +262,12 @@ public function serialize()
       array(QubitTaxonomy::PLACE_ID, QubitTaxonomy::SUBJECT_ID)
     );
 
+    // Related objects
+    if (count($relations = $this->serializeObjectRelations()))
```

Comment:
In this case, where the count is not actually needed, `!empty()` is a bit faster.